Lib/fontTools/otlLib/builder.py

```diff
@@ -46,6 +48,162 @@ def buildLookup(subtables, flags=0, markFilterSet=None):
              "LOOKUP_FLAG_USE_MARK_FILTERING_SET; flags=0x%04x" % flags)
     return self
 
+def optimizeChainingFormats(subtables, builder):
+
+    def binary_len(st):
```

Comment:
This file seems to be otherwise consistent in its use of camlCase function names.